config/config.toml

```diff
@@ -156,6 +156,9 @@ retryTimeWindowInMins = 180
 minRetryBackoffInS = 10
 maxRetryBackoffInS = 300
 allowAbortedUserJobsCountForProcessing=1
+# currently these configs are supported destination(type) wise
```

Comment:
Instead of a comment here, maybe we should put a comment above/next to all such configs. It will be easy to read.

Eg.
```
// Destination type config support
allowAbortedUserJobsCountForProcessing = 1 

```

Reply:
Also, there is a variable `ser` that we are not using anymore, let's remove that as part of this PR.

Reply:
done